src/socket.cpp

```diff
@@ -70,7 +70,7 @@ CSocket::CSocket ( CServer* pNServP, const quint16 iPortNumber, const quint16 iQ
     QObject::connect ( this, &CSocket::ProtocolCLMessageReceived, pServer, &CServer::OnProtocolCLMessageReceived );
 
     QObject::connect ( this,
-                       static_cast<void ( CSocket::* ) ( int, int, CHostAddress )> ( &CSocket::NewConnection ),
+                       static_cast<void ( CSocket::* ) ( int, int, const CHostAddress& )> ( &CSocket::NewConnection ),
```

Comment:
Mmm, not sure on this one, either.  Aren't we creating a new entity here and throwing away the source?  Is it okay to have a dangling reference?

Reply:
I will check, but I believe it's ok.

Reply:
I changed that one back to call by value already, as it is an implicit signal-slot connection.